bench/single_file_bench.exs

```diff
@@ -1,27 +0,0 @@
-defmodule Dogma.SingleSourceBench do
```

Comment:
TBH I removed it because it seemed broken and it was easier to delete than fix :smile: If it's useful I could spend some time fixing it.